examples/digital_fingerprinting/production/morpheus/benchmarks/conftest.py

```diff
@@ -64,24 +64,25 @@ def pytest_benchmark_update_json(config, benchmarks, output_json):
         elif "message_path" in PIPELINES_CONF[bench["name"]]:
             source_message_glob = path.join(curr_dir, PIPELINES_CONF[bench["name"]]["message_path"])
             for message_fn in glob.glob(source_message_glob):
-                message_file = open(message_fn)
-                control_message = json.load(message_file)
+                with open(message_fn, encoding='utf-8') as message_file:
+                    control_message = json.load(message_file)
                 inputs = control_message.get("inputs")
                 # Iterating over inputs
-                for input in inputs:
+                for inpt in inputs:
```

Comment:
> for inpt in inputs:
Is renaming `input` to `inpt` intentional?

Reply:
@dagardner-nv yeah, that's intentional. Input is a builtin function.